### PR TITLE
Define a module for managing E-Series top-level settings

### PR DESCRIPTION
**Pull request by lmprice**
##### SUMMARY

There are multiple settings that are defined at a global level for
E-Series systems, but don't necessarily fit with anything else. This
module is intended to provide a place to encapsulate those.

##### ISSUE TYPE

New Module Pull Request

##### COMPONENT NAME
netapp_e_global

##### ANSIBLE VERSION

```
ansible 2.6.0 (detached HEAD ce8c336d52) last updated 2018/04/18 11:23:03 (GMT -500)
  config file = None
  configured module search path = [u'/home/lorenp/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/lorenp/ansible/lib/ansible
  executable location = /home/lorenp/ansible/bin/ansible
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by lmprice** [comment]
@amit0701 @carchi8py @hulquest @mattclay @ndswartz @schmots1 Can you give me a review here?

**Review by ndswartz — APPROVED** [review]
shipit

File: lib/ansible/utils/module_docs_fragments/netapp.py [hunk 1]
```diff
@@ -121,24 +121,31 @@ class ModuleDocFragment(object):
   api_username:
     required: true
     description:
-    - The username to authenticate with the SANtricity WebServices Proxy or embedded REST API.
+    - The username to authenticate with the SANtricity Web Services Proxy or Embedded Web Services API.
```
Comment (gundalow) [inline lib/ansible/utils/module_docs_fragments/netapp.py]: I think this change needs removing from this PR (as it's been done in another PR which has since been merged)

Also you need to rebase to resolve the merge conflict https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html

File: test/sanity/validate-modules/ignore.txt [hunk 2]
```diff
@@ -1207,7 +1207,6 @@ lib/ansible/modules/storage/netapp/na_cdot_volume.py E322
 lib/ansible/modules/storage/netapp/na_cdot_volume.py E324
 lib/ansible/modules/storage/netapp/na_cdot_volume.py E325
 lib/ansible/modules/storage/netapp/netapp_e_amg.py E322
```
Comment (gundalow) [inline test/sanity/validate-modules/ignore.txt]: I think this change needs removing from this PR (as it's been done in another PR which has since been merged)

Also you need to rebase to resolve the merge conflict https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html

**Review by gundalow — APPROVED** [review]
Module itself looks OK, please just remove the docs_fragment and ignore.txt files from this PR and then rebase your PR I think this change needs removing from this PR (as it's been done in another PR which has since been merged)

Also you need to rebase to resolve the merge conflict https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html
